queue-workers/bull-queue-workers/worker.js

```diff
@@ -54,4 +54,4 @@ worker.on('failed', (job, error) => {
 });
 
 console.log('BullMQ worker started');
-logger.info(`BullMQ worker started at ${new Date().toISOString()} on ${process.env.ENV}/${process.platform}.`);
\ No newline at end of file
+logger.info(`BullMQ worker started at ${new Date().toISOString()} on ${process.env.ENV}/${process.platform}.`, 'WORKER-STARTED');
```

Comment:
I am a bit skeptical about logging so much information about the worker.
We already separated the prod Logtail from dev so logging the `ENV` again feels redundant. Also logging the platform feels like a security risk.
We could just have ` logger.info('Worker started', 'WORKER-STARTED')` to make it generic